packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -124,8 +124,7 @@ export default ({
   if (metastring && codeBlockTitleRegex.test(metastring)) {
     // Tested above
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    codeBlockTitle = metastring
-      .match(codeBlockTitleRegex)![0];
+    codeBlockTitle = metastring.match(codeBlockTitleRegex)![0];
```

Comment:
Change ` [0]` to `[1]` if PR 3685 is accepted, undo this change if not

Reply:
this is done automatically be merging